src/containers/task-management/task-list-view.tsx

```diff
@@ -240,10 +265,54 @@ export class TaskListView extends React.Component<RouteComponentProps, IState> {
           <DateComponent date={finished_at} />
         </td>
         {this.statusLabel({ state })}
+        {this.cancelButton(state, item)}
```

Comment:
```suggestion
        <td>{this.statusLabel({ state })}</td>
        <td>{this.cancelButton(state, item)}</td>
```

and remove `<td>` from those functions please :)

So that `statusLabel` outputs a `Label` (or nothing), and `cancelButton` outputs a `Button` (or nothing), instead of the table logic leaking inside those.